build/build.proj

```diff
@@ -151,7 +151,7 @@
     
   </Target>
 
-  <Target Name="Test" Condition="'$(BuildTemplates)' != 'true'">
+  <Target Name="Test" Condition="'$(BuildTemplates)' != 'true' and '$(RunTests)' == 'true'">
```

Comment:
There's already a `BuildWithoutTesting` target in this file, I would prefer to invoke that target instead of `Build` in the Build.ps1 script if RunTests is false.

Note that for this to work you'll need to change the invocation from `dotnet build` to `dotnet msbuild` as the build command currently [always runs the `Build` target](dotnet/cli#4815).